src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -238,27 +266,32 @@ def collect_tool_function_in_module(m):
         return tool_methods[0]
 
 
-def generate_python_tool(name, content, source=None):
+def generate_python_tool_meta_dict(name, content, source=None):
     m = load_python_module(content, source)
     f, initialize_inputs = collect_tool_function_in_module(m)
-    tool = _parse_tool_from_function(f, initialize_inputs=initialize_inputs)
+    tool = _parse_tool_from_function(
+        f, initialize_inputs=initialize_inputs, gen_custom_type_conn=True, skip_prompt_template=True
```

Comment:
Why set the `gen_custom_type_conn` and `skip_prompt_template` to true? To my understand, these should be false for script tool.

Reply:
revert it